src/app/layout/header/header.component.html

```diff
@@ -1,13 +1,16 @@
+<div class="fill-space;" class="header-accent-bar"></div>
 <mat-toolbar color="primary" *ngIf="isAuthenticated && !isVerificationRequired">
-  <button class="mat-button menu-button" (click)="toggleSidebar()" *ngIf="isMobile">
+  <button class="menu-button" (click)="toggleSidebar()" *ngIf="isMobile && !isMenuShowing">
     <mat-icon class="clickable menu-icon">menu</mat-icon>
   </button>
+  <button class="menu-button" (click)="toggleSidebar()" *ngIf="isMobile && isMenuShowing">
+    <mat-icon class="clickable menu-icon">clear</mat-icon>
+  </button>
   <span routerLink="/" class="clickable">
-    <span class="nav-item-container" style="color:white;">
-      Sample App
-    </span>
   </span>
   <span class="fill-space"></span>
-  <app-notifications-icon></app-notifications-icon>
-  <app-profile-icon></app-profile-icon>
+  <span class="balance-display" *ngIf=(!isMenuShowing)>
+    ${{ totalBalance }} <br/><span class="balance-label">BALANCE</span>
```

Comment:
Instead of $ you can use the currency pipe in angular. {{ totalBalance | currency }}